packages/yargs/index.js

```diff
@@ -5,6 +5,8 @@ const createYargs = require('yargs');
 
 const { bootstrap } = require('@untool/core');
 
+exports.Mixin = require('./mixin.core');
```

Comment:
What's the reason for exporting the mixin?
Also what whould happen in a case where I have multiple mixins (core, browser, server)?